src/config/db.js

```diff
@@ -1,15 +1,13 @@
-require('dotenv').config(); // Load environment variables from .env file
+require('dotenv').config();
```

Comment:
I think we don't need this file anymore, there is a different configuration for database